crypto/encode_decode/encoder_pkey.c

```diff
@@ -210,10 +210,11 @@ static void encoder_destruct_EVP_PKEY(void *arg)
 static int ossl_encoder_ctx_setup_for_EVP_PKEY(OSSL_ENCODER_CTX *ctx,
                                                const EVP_PKEY *pkey,
                                                int selection,
-                                               OSSL_LIB_CTX *libctx,
                                                const char *propquery)
 {
     struct construct_data_st *data = NULL;
+    const OSSL_PROVIDER *prov = EVP_KEYMGMT_provider(pkey->keymgmt);
```

Comment:
Static analyser will not like derefs on the pkey
followed by a check for pkey == NULL.

Reply:
Ah, good point!

Reply:
Fixed, I think

Reply:
You fixed one of them..
I was talking about the code in ossl_encoder_ctx_setup_for_EVP_PKEY() :)

Reply:
Damn, did I miss that one?  Up I did.  Fixed!